database/mysql/mysql.go

```diff
@@ -93,13 +98,39 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 		config: config,
 	}
 
+	if err := mx.setupDefaultConfig(); err != nil {
+		return nil, err
+	}
+
 	if err := mx.ensureVersionTable(); err != nil {
 		return nil, err
 	}
 
 	return mx, nil
 }
 
+func (m *Mysql) setupDefaultConfig() error {
```

Comment:
Can you move all of this logic to `WithConnection()`. Then inside `WithInstance()` we can call `WithConnection()` and set `m.db = instance`.

Reply:
Aha, I considered both designs and I seem to have picked the wrong one :sob: 
I'll change this ASAP, thanks for your review.

Reply:
Done!
By the way, I just reminded why I chose the previous pattern: It's because now I need `WithConnection` to return a `*Mysql` type instead of `database.Driver`.

Reply:
Why do you need a `*Mysql`?

Reply:
Because `WithInstance` needs to set `mx.db`